tools/ssvm-aot/ssvmr.cpp

```diff
@@ -146,12 +130,16 @@ int main(int Argc, const char *Argv[]) {
         break;
       }
     }
+    if (FuncType.Params.size() + 1 < Args.value().size()) {
+      for (size_t I = FuncType.Params.size() + 1; I < Args.value().size();
+           ++I) {
+        const uint64_t Value = std::stoll(Args.value()[I]);
+        FuncArgs.emplace_back(Value);
+      }
+    }
 
-    if (auto Result = VM.execute(FuncName, FuncArgs); !Result) {
-      std::cout << FuncName << " failed. Error code : "
-                << static_cast<uint32_t>(Result.error()) << '\n';
-      return EXIT_FAILURE;
-    } else {
+    if (auto Result = VM.execute(FuncName, FuncArgs)) {
```

Comment:
Missing `!Result`?

Reply:
This is success case. Print the return values.
The fail case will return EXIT_FALURE directly because SSVM will log error messages.

Reply:
Okay.